src/calculateRentalCost.js

```diff
@@ -5,6 +5,25 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let costOfrent = 40 * days;
+  const longTermThreshold = 7;
+  const midTermThreshold = 3;
+  const longTermDiscount = 50;
+  const midTermDiscount = 20;
+
+  if (days >= longTermThreshold) {
+    costOfrent = costOfrent - longTermDiscount;
+
+    return costOfrent;
```

Comment:
You can remove the assignment in line 15 and directly return the discounted price from within the `if` statement. This simplifies the function and adheres to checklist item 4, preferring `if` with `return` to simplify later conditions.